### DISCUSSION TITLE
10/24 김도연 블로그 생성

### DISCUSSION THREAD

**Discussion by Devkdy22** [Orm-camp/blog-feedback#22] [동글동글 3기 피드백]
[https://velog.io/@devkdy99/posts](url)
저는 벨로그로 블로그를 생성하였습니다.
기술적으로 배운 내용들이나, 경험한 상황들을 해결한 내용들 블로그에 작성하면서 운영하려고 합니다.
매주,, 하나의 글을 써보려고 노력해보려합니다..! 🥹

**Comment by kimjigyun** [comment]
나중에 블로그 보러 오겠습니다 !!
